spring-boot-project/spring-boot-docs/src/docs/asciidoc/getting-started/first-application.adoc

```diff
@@ -141,9 +141,10 @@ Open your favorite text editor and add the following:
 	plugins {
 		id 'java'
 		id 'org.springframework.boot' version '{spring-boot-version}'
-		id 'io.spring.dependency-management' version '1.1.0'
 	}
 
+	apply plugin: 'io.spring.dependency-management'
```

Comment:
Does this work without defining a `classpath` dependency in `buildscript`?
Even if it does, I might keep it as is (despite of I'm not a fan of using `id`) to keep it consistent with the [docs](https://docs.spring.io/dependency-management-plugin/docs/current/reference/html/#getting-started) of the plugin and the output of [initializr](https://start.spring.io/).

Reply:
@jonatan-ivanov I thought it's a better way as we can rely on `org.springframework.boot` Gradle plugin managed version this way.

See https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/#getting-started

Reply:
Interesting. The Gradle plugin documentation recommends something different than start.spring.io uses.

@wilkinsona What do you think what we should put in the docs?

Reply:
I think we should make this change so that Boot's documentation is consistent with itself at least. I can't recall why start.spring.io is different but I think that can be dealt with separately.

Reply:
fyi: https://github.com/spring-io/initializr/issues/1428